src/client/Microsoft.Identity.Client/AuthScheme/TokenType.cs

```diff
@@ -32,6 +32,12 @@ internal enum TokenType
         /// Extension token type.
         /// Extension = 5
         /// </summary>
-        Extension = 5
+        Extension = 5,
+
+        /// <summary>
+        /// MTLS Pop token type.
+        /// MTLS_Pop = 6
+        /// </summary>
+        Mtls_Pop = 6
```

Comment:
```suggestion
        MtlsPop = 6
```

Reply:
I chose mtls_pop as that is excatly what we pass to ESTS as token type, is there a reason to change it?